CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorConfigurationRequestHandler.swift

```diff
@@ -22,6 +22,79 @@ public class ReplicatorConfigurationRequestHandler {
+            let channels: [String]? = args.get(name: "channels")
+            let documentIDs: [String]? = args.get(name: "documentIDs")
+            var config = CollectionConfiguration()
+            if(conflictResolver != nil) {
+                config.conflictResolver = conflictResolver
+            }
+            if pull_filter != false {
+                if filter_callback_func == "boolean" {
+                    config.pullFilter = _replicatorBooleanFilterCallback;
+                } else if filter_callback_func == "deleted" {
+                    config.pullFilter = _replicatorDeletedFilterCallback;
+                } else if filter_callback_func == "access_revoked" {
+                    config.pullFilter = _replicatorAccessRevokedCallback;
+                } else {
+                    config.pullFilter = _defaultReplicatorFilterCallback;
+                }
+            }
+            if push_filter != false {
+                if filter_callback_func == "boolean" {
+                    config.pushFilter = _replicatorBooleanFilterCallback;
+                } else if filter_callback_func == "deleted" {
+                    config.pushFilter = _replicatorDeletedFilterCallback;
+                } else if filter_callback_func == "access_revoked" {
+                    config.pushFilter = _replicatorAccessRevokedCallback;
+                } else {
+                    config.pushFilter = _defaultReplicatorFilterCallback;
+                }
+            }
+            if(channels != nil) {
+                config.channels = channels
+            }
+            if(documentIDs != nil){
+                config.documentIDs = documentIDs
+            }
+            return config
+        
+        case "replicatorCollection_addCollection":
+            var replicatorConfiguration: ReplicatorConfiguration = args.get(name: "replicatorConfiguration")!
+            let collection: Collection = args.get(name: "collections")!
+            let collectionConfiguration: CollectionConfiguration? = args.get(name: "configuration")
+            if(collectionConfiguration != nil) {
+                replicatorConfiguration.addCollection((collection), config: collectionConfiguration)
+            }
+            else {
+                replicatorConfiguration.addCollection((collection))
+            }
+        
+        case "replicatorCollection_addCollections":
+            var replicatorConfiguration: ReplicatorConfiguration = args.get(name: "replicatorConfiguration")!
+            let collection: [Collection] = args.get(name: "collections")!
+            let collectionConfiguration: CollectionConfiguration? = args.get(name: "configuration")
+            if(collectionConfiguration != nil) {
+                replicatorConfiguration.addCollections((collection), config: collectionConfiguration)
+            }
+            else {
+                replicatorConfiguration.addCollections((collection))
+            }
```

Comment:
any reasons for this, why not 
```suggestion
            replicatorConfiguration.addCollections((collection), config: collectionConfiguration)
```